pkgs/by-name/as/aseprite/package.nix

```diff
@@ -167,7 +166,8 @@ clangStdenv.mkDerivation (finalAttrs: {
                 - Pixel-art specific tools like filled Contour, Polygon, Shading mode, etc.
                 - Onion skinning.
     '';
-    maintainers = [
+    maintainers = with lib.maintainers; [
+      iamanaws
```

Comment:
Please, move this in a separate commit (`aseprite: add iamanaws to maintainers`).